client/src/components/participant-details/track-graduation.js

```diff
@@ -2,25 +2,39 @@ import { Grid, Typography, Dialog } from '@material-ui/core';
 import { Button } from '../../components/generic/Button';
 
 import { ManageGraduationForm } from '../modal-forms/ManageGraduationForm';
-import React, { useState } from 'react';
+import { AssignCohortForm } from '../modal-forms/AssignCohort';
+import React, { useEffect, useState } from 'react';
 import moment from 'moment';
 import { createPostHireStatus } from '../../services/participant';
 export const TrackGraduation = (props) => {
-  const cohort = props?.participant?.cohort;
+  const [cohort, setCohort] = useState(null);
   const { fetchData } = props;
   const [showEditModel, setShowEditModal] = useState(false);
+
+  useEffect(() => {
+    setCohort(props.participant?.cohort);
+  }, [setCohort, props.participant?.cohort]);
   return (
     <>
       <Grid container>
-        <Grid item xs={4}>
+        <Grid item xs={3}>
+          <Typography>Cohort Name</Typography>
```

Comment:
I think we should either remove "Name" from here, or lowercase it to match other copy styling